pages/docs/advanced/react-native.en-US.mdx

```diff
@@ -96,3 +96,64 @@ import { AppState } from 'react-native'
 ```
 
 For `initReconnect`, it requires some 3rd party libraries such as [NetInfo](https://github.com/react-native-netinfo/react-native-netinfo) to subscribe to the network status. The implementation will be similar to the example above: receiving a `callback` function and trigger it when the network recovers from offline, so SWR can start a revalidation to keep your data up-to-date.
+
+
+### AsyncStorage Based Persistent Cache
+
+You might want to sync your cache to `asyncStorage`. Here's an example implementation:
+
+```jsx
+const useAsyncStorage = () => {
+  const [cache, setCache] = useState(null)
+
+  useLayoutEffect(() => {
```

Comment:
The example looks good, just 1 question: is there any reason use `useLayoutEffect` here instead of `useEffect`?

Reply:
yep, you're right, thanks for your feedback
I must change this to useEffect

Reply:
yep, you're right
Thanks for the feedback, I'm gonna update this one @huozhi